packages/core/src/components/sections/ProductDetails/ProductDetails.tsx

```diff
@@ -223,6 +226,17 @@ function ProductDetails({
                 )
               }
               refNumber={showRefNumber && productId}
+              reviewsAndRating={{
+                ratingValue: apiConfig.reviewsAndRatings
+                  ? rating.average
+                  : undefined,
+                reviewsCount: apiConfig.reviewsAndRatings
+                  ? rating.totalCount
+                  : undefined,
+                noReviewsText,
+                multipleReviewsText,
+                singleReviewText,
+              }}
```

Comment:
suggestion: only send when the feature is activated
```suggestion
              {...(apiConfig.reviewsAndRatings && {
                reviewsAndRating: {
                  ratingValue: rating.average,
                  reviewsCount: rating.totalCount,
                  noReviewsText,
                  multipleReviewsText,
                  singleReviewText,
                },
              })}
```